python/lsst/ts/watcher/rules/telemetry.py

```diff
@@ -0,0 +1,174 @@
+                poll_names=[],
+            ),
+            RemoteInfo(
+                name=remote_name,
+                index=remote_index,
+                callback_names=["evt_summaryState"],
+                poll_names=[],
+            ),
+        ]
+        super().__init__(
+            config=config,
+            name=f"Telemetry.{remote_name}:{remote_index}",
+            remote_info_list=remote_info,
+            log=log,
+        )
+        self.telemetry_timer_task = utils.make_done_future()
+        self.csc_should_receive_telemetry = False
+
+    @classmethod
+    def get_schema(cls):
+        # NOTE: another option is to have separate time limits for
+        # warning, serious and critical. But this requires up to 3 timers
+        # for each CSC, which adds many more tasks that the CSC has to manage.
+        schema_yaml = f"""
+            $schema: 'http://json-schema.org/draft-07/schema#'
+            description: Configuration for Telemetry.
+            type: object
+            properties:
+                name:
+                    description: >-
+                        CSC name and index in the form `name` or `name:index`.
+                        The default index is 0.
+                    type: string
+                callback_name:
+                    description: >-
+                        The name of the telemetry topic to monitor.
+                    type: string
+                timeout:
+                    description: Maximum allowed time between telemetry (sec).
+                    type: number
+                    default: 15
+                alarm_severity:
+                    description: >-
+                        Alarm severity if the time is exceeded. One of:
+                        * {AlarmSeverity.WARNING.value} for warning
+                        * {AlarmSeverity.SERIOUS.value} for serious
+                        * {AlarmSeverity.CRITICAL.value} for critical
+                    type: integer
+                    enum:
+                    - {AlarmSeverity.WARNING.value}
+                    - {AlarmSeverity.SERIOUS.value}
+                    - {AlarmSeverity.CRITICAL.value}
+                    default: {AlarmSeverity.CRITICAL.value}
+            required:
+            - name
+            - callback_name
+            - timeout
+            - alarm_severity
+            additionalProperties: false
+        """
+        return yaml.safe_load(schema_yaml)
+
+    def compute_alarm_severity(
+        self, data: salobj.BaseMsgType, **kwargs: typing.Any
+    ) -> AlarmSeverityReasonType:
+        topic_callback = kwargs["topic_callback"]
+        _, _, topic_name = topic_callback.topic_key
+        if topic_name == "evt_summaryState":
+            self.log.debug(
+                f"Received evt_summaryState for {self.name} with summaryState={data.summaryState}"
+            )
+            self.csc_should_receive_telemetry = data.summaryState in [
+                salobj.State.DISABLED,
```

Comment:
Note that not all CSCs publish telemetry when they are in DISABLED. You might want to make this configurable.